### PR TITLE
fix routes

### PR DESCRIPTION
**Pull request by mindrunner**
Fixes problem with routing after refactoring code

### CODE REVIEW

**Comment by Madkonix** [comment]
@AyFarukh @mindrunner Going through the pull request and I see this:
image

What do you guys think of:
1. changing the credentials (these are already compromised)
2. update the code to use variables instead (just like for the port)
3. more of an open question: why connect to a db in the cloud if we are already running a docker for mongodb ? Seems redundant one of them got to go.

**Comment by mindrunner** [comment]
Yes, you are absolutely right. 

1) Should be done ASAP
2) Happy to see a PR. Let us pass the connection string in by environment
3) This is a question for our DevOps guy! :)

**Comment by AyFarukh** [comment]
I will do it in couple of hours.

**Comment by Madkonix** [comment]
1. Done and tested, thanks @AyFarukh 
2. @mindrunner that would mean merging this one first then overriding it with another commit. Better kill 2 birds with one stone, so instead I will make the changes then submit a change request to your pull request. Ok ?
3. Sure, but still better have other points of view and perspectives before making the decision. I  say let's drop the cloud and use the local mongodb docker with persistent volume.

File: app/routes/index.js [hunk 1]
```diff
@@ -1,29 +1,25 @@
...
+    app.get(prefix + '/country_List', homeServices.get_country);
+    app.get(prefix + '/country', homeServices.get_all_country);
+    app.get(prefix + '/state', homeServices.get_all_state);
+    app.get(prefix + '/Check_Url', homeServices.Check_Url);
 }
 mongoose.connect(
-  'mongodb+srv://Farukh:110581F.A@rest-6ss50.mongodb.net/test?retryWrites=true&w=majority',
-  { useNewUrlParser: true, useUnifiedTopology: true },
-  ()=> console.log("Connected to db!")
+    'mongodb+srv://Farukh:110581F.A@rest-6ss50.mongodb.net/test?retryWrites=true&w=majority',
```
Comment (Madkonix) [inline app/routes/index.js]: There is already an env variable for this, just need to use it. To do so, just replace this line with`  process.env.MONGO_URI,`

**Comment by Madkonix** [comment]
@mindrunner I tried running the app after applying the patch of this pr but I still run into the same issue (504 - Gateway timeout), do you confirm it works on your local ?

**Comment by mindrunner** [comment]
image

Working all fine here. Maybe be more precise about what is not working?

**Comment by mindrunner** [comment]
pushed another fix. all routes should work.

See https://shopiloc.lukius.de/ where this branch is deployed

**Pull request merged**